packages/developer-portal/src/components/pages/login/login.tsx

```diff
@@ -54,8 +53,8 @@ export const Login: React.FunctionComponent<LoginProps> = () => {
     dispatch,
   ])
 
-  if (hasSession) {
-    const redirectRoute = getDefaultPath(isDesktopMode, isFirtTimeLogin)
+  if (connectSession) {
```

Comment:
We no longer need to re-direct users if they have a session so this block can be removed